include/mc_rtc/Schema.h

```diff
@@ -67,6 +67,28 @@ inline constexpr bool is_std_vector_schema_v = []()
   else { return false; }
 }();
 
+/** Type-trait to detect an std::map */
+template<typename T>
+struct is_std_map : std::false_type
+{
+};
+
+template<typename... Args>
+struct is_std_map<std::map<Args...>> : std::true_type
+{
+};
```

Comment:
This will accept any map but `mc_rtc::Configuration` only accepts `std::map<std::string, T>` (this is a JSON limitation) so the loading/saving code would fail for any other key type.